source/transactions/tests/README.rst

```diff
@@ -268,3 +291,59 @@ afterClusterTime
 A ``readConcern.afterClusterTime`` value of ``42`` in a command-started event
 is a fake cluster time. Drivers MUST assert that the actual command includes an
 afterClusterTime.
+
+Mongos Pinning Prose Tests
+==========================
+
+The following tests ensure that a ClientSession is properly unpinned after
+a sharded transaction. Initialize these tests with a MongoClient connected
+to multiple mongoses.
+
+These tests use a cursor's address field to track which server an operation
+was run on. If this is not possible in your driver, use command monitoring
```

Comment:
Out of curiosity, if it's possible to write these using command monitoring why not write them as YAML tests?

Reply:
Oh, missed Jesse's comment above. I'm guessing it's not prose because we don't have an easy way to specify a loop in the test runner format. Resolving the comment.